### PR TITLE
Tests static

### PR DESCRIPTION
**Pull request by meyer28j**
Created directed graphs for path coverage and data flow coverage analysis of getProjectileDrag(). In Testing.md, I explained the process of each type of analysis. The results, as we might expect from a straightforward function like this, is that all the test requirements for the data flow DU-Path Coverage are the same as the test requirements for the Prime Path Coverage. Note that this is expected as Prime Path Coverage subsumes DU-Path Coverage. Also note that in this case, the Prime Path Coverage also happens to be Complete Path Coverage.

### CODE REVIEW

File: docs/testing.md
Comment (katanallama) [inline docs/testing.md:1]: Could you copy this over to res/testing.org so the export doesn't break?

The TOC and IDs will be auto generated at export to .md

File: docs/res/graph-data-flow.jpg
Comment (katanallama) [inline docs/res/graph-data-flow.jpg:1]: LGTM

File: docs/res/graph-structural.jpg
Comment (katanallama) [inline docs/res/graph-structural.jpg:1]: LGTM

File: src/test/java/com/pss/handlers/GetProjectileDragPathTest.java
Comment (katanallama) [inline src/test/java/com/pss/handlers/GetProjectileDragPathTest.java:1]: Seeing this now we can probably refactor all the unit testing into one file assuming BVD is going to function in the same way. Dan's interface tests can probably live here too as an additional function. 

No need to change now, we can refactor once all the tests are written.

**Review by katanallama — CHANGES_REQUESTED** [review]
Just a quick change, move testing.md contents into res/testing.org, will re-export it when we go to refactor.

File: docs/testing.md
Comment (meyer28j) [reply docs/testing.md:1]: Done

**Review by katanallama — APPROVED** [review]

**Pull request merged**